### ISSUE TITLE
Intermittent pass in /html/browsers/browsing-the-web/history-traversal/browsing_context_name.html

### ISSUE THREAD

**Issue by simartin** [servo/servo#12568]
Found during a test run for https://github.com/servo/servo/pull/12565

```
Tests with unexpected results:
  ▶ Unexpected subtest result in /html/browsers/browsing-the-web/history-traversal/browsing_context_name.html:
  └ PASS [expected FAIL] Retaining window.name on history traversal
```

**Comment by KiChjang** [comment]
Not an intermittent.

**Comment by jdm** [comment]
cc @ConnorGBrewster

**Comment by ConnorGBrewster** [comment]
Possibly fixed by #12563

**Comment by ConnorGBrewster** [comment]
I think it might be worth disabling this for now, since we don't even have browsing context/window names implemented yet. I am sure this is just another issue with timing.

**Comment by jdm** [comment]
This was disabled.

**Comment by frewsxcv** [comment]
Blocked on https://github.com/servo/servo/issues/14453.

**Comment by mrobinson** [comment]
This test is no longer disabled.
